zip-0243.rst

```diff
@@ -54,7 +52,8 @@ A new transaction digest algorithm is defined::
    11. valueBalance of the transaction (8-byte little endian)
    12. sighash type of the signature (4-byte little endian)
 
-   13. If we are serializing an input (i.e. this is not a JoinSplit signature hash):
+   13. If we are serializing a transparent input (i.e. this hash is not for a JoinSplit signature,
+       Spend authorization signature, or binding signature):
```

Comment:
I don't think it's clear to the reader what a binding signature is - I assume you mean the one of the commitment sums; I'm not sure the reader knows what Spend authorization signature is either, though that's easier to understand.

Reply:
These are defined in the protocol spec.